### ISSUE TITLE
Oracle dialect fails to detect / (slash) as a statement terminator

### ISSUE THREAD

**Issue by saulotoledo** [sqlfluff/sqlfluff#7497]
### Search before asking

- [x] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.

### What Happened

The Oracle dialect fails to correctly identify and parse the / (slash) symbol when used as a statement terminator. In Oracle SQL and PL/SQL, the slash is the standard execution character used to terminate and execute blocks of code (such as CREATE PROCEDURE, CREATE TRIGGER, or anonymous blocks) within a script. Currently, SQLFluff does not recognize this as a valid delimiter, leading to parsing errors in multi-statement scripts.

### Expected Behaviour

SQLFluff should recognize the / character as a valid statement terminator, particularly for PL/SQL blocks, allowing multiple blocks to be defined and terminated sequentially within a single file.

### Observed Behaviour

The parser fails to recognize the `/` as a delimiter. Internal logic within `DelimiterGrammar` appears to be incorrectly defined as a `Sequence` that requires both a semicolon and a slash, or it simply fails to include the slash as a standalone alternative to the semicolon.

### How to reproduce

1. Create repro.sql:

```sql
-- This fails to parse the slash delimiter between blocks
CREATE PROCEDURE my_proc IS 
BEGIN 
    NULL; 
END;
/

BEGIN 
    NULL; 
END;
/
```

Run linting:

```bash
sqlfluff lint repro.sql --dialect oracle
```

### Dialect

oracle

### Version

4.0.4

### Configuration

Standard default configuration using `--dialect oracle`.

### Are you willing to work on and submit a PR to address the issue?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)
